### PR TITLE
Added contributors page | Modified Sections and readme.md

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- I have added many modifications which I felt would help and are needed from my slides. Head over to files changed for a complete summary. 
- Made and modified some new sections [Contributors, Links for more info and Where to go from here]
- Improved Indentation through Atom-Beautify (Though I might agree at some places it might made mistakes. )
- Added License info in README.md and made it a little more descriptive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # git-intro
 Learning version control with git
 
-This is currently under development.
```
Comment (CuriousLearner) [inline README.md]: This is still under development.

File: index.html [hunk 2]
```diff
@@ -33,57 +34,62 @@
...
                 <p>
                     <small>
-                        Created by
+                        Created by:
                             <a href="http://www.SanyamKhurana.com">Sanyam Khurana</a> /
                             <a href="http://twitter.com/ErSanyamKhurana">@ErSanyamKhurana</a> /
-                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a>
+                            <a href="http://github.com/CuriousLearner">@CuriousLearner</a> /
+                            <a href="https://thegeekyway.com">The Geeky Way</a><br>
+                        Edited by:
```
Comment (CuriousLearner) [inline index.html]: Move it to contributors slide, including all the people who've contributed.

File: index.html [hunk 3]
```diff
@@ -33,57 +34,62 @@
...
             <section id="fr1">
                 <h2>Daily Tasks as Programmer</h2>
                 <p>
-                <ul>
-                    <li class="fragment">Create “Things”</li>
-                    <li class="fragment">Save “Things”</li>
-                    <li class="fragment">Edit “Things”</li>
-                    <li class="fragment">Save “Things” AGAIN!</li>
-                </ul>
+                    <ul>
```
Comment (CuriousLearner) [inline index.html]: Let's use spaces here instead of tab. 

let's use it everywhere.

File: index.html [hunk 4]
```diff
@@ -33,57 +34,62 @@
...
+                    </ul>
                 </p>
             </section>
             <section id="fr3">
                 <h2>Version Control comes to rescue</h2>
                 <section>
                     <ul>
                         <li class="fragment">Method to track changes to file(s) and folder(s) and your binaries.</li>
-                        <li class="fragment">Who changed what and why? - via commit messages</li>
+                        <li class="fragment">Who changed what and why? - via commit messages.</li>
```
Comment (CuriousLearner) [inline index.html]: Add, who changed what, why and when.

File: index.html [hunk 5]
```diff
@@ -103,65 +109,60 @@ <h2>Version Control comes to rescue</h2>
...
-                            is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based
-                            on the contents of a file or directory structure in Git. A SHA-1 hash looks something like this:
-                            <pre><code class="hljs" data-trim
-                                       contenteditable>24b9da6552252987aa493b52f8696cd6d3b00373</code></pre>
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">It compresses the .git contents</li>
+                            <li class="fragment">It has integrity</li>
+                            <li class="fragment">The mechanism that Git uses for this checksumming is called a SHA-1 hash. This is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based on the contents of a file or directory structure
```
Comment (CuriousLearner) [inline index.html]: Trim these lines as these were before, and do this in other parts too which you've touched. Although IDEs fix this, but let's ensure that even if I'm opening it on a simple text editor, `nano`, `vim`, `TextEdit` or whatever, I could see everything without doing a horizontal scroll.

File: index.html [hunk 6]
```diff
@@ -103,65 +109,60 @@ <h2>Version Control comes to rescue</h2>
...
                 <section>
                     <p>
-                    <ul>
-                        <li class="fragment">You will see these hash values all over the place in Git because it uses them
-                            so much. In fact, Git stores everything in its database not by file name but by the hash value
-                            of its contents.
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">You will see these hash values all over the place in Git because it uses them so much. In fact, Git stores everything in its database not by file name but by the hash value of its contents.
```
Comment (CuriousLearner) [inline index.html]: Trim this.

File: index.html [hunk 7]
```diff
@@ -207,64 +208,91 @@ <h2>Git Workflow</h2>
...
+                        <ul>
+                            <li class="fragment">Pushing to a repository requires authentication that needs to be provided in the form of your Github username and password.</li>
+                        </ul>
+                    <p>
                 </section>
                 <section>
-                <span>
-                    Repeat till you get them right ... <br> Then move on to more advanced topics such as branching, rebasing and cherry-picking commits.
-                </span>
+                    <span>
```
Comment (CuriousLearner) [inline index.html]: Indentation is odd here.

File: index.html [hunk 8]
```diff
@@ -207,64 +208,91 @@ <h2>Git Workflow</h2>
...
+                        <li>Widen your network and find fun new side-projects.</li>
+                        <li>Use your GitHub profile to showcase projects on your resume.</li>
+                    </ul>
+                </p>
+            </section>
+            <section>
+                <h2>Links for learning more</h2>
+                <p>
+                    <ul>
+                        <li>Documentation for reference - <a href="http://www.git-scm.com/">http://www.git-scm.com/</a> </li>
```
Comment (CuriousLearner) [inline index.html]: Including too many links for beginners would make them go absolutely no where. You can mention them as Resources, but keep one slide as it is, which was, "where to go from here" --- And say git-scm.

File: index.html [hunk 9]
```diff
@@ -207,64 +208,91 @@ <h2>Git Workflow</h2>
...
+                        <li>Sandeep Choudhary - available on Github by <a href="https://github.com/Skchoudhary">@Skchoudhary</a></li>
+                        <li>Vishak Sharma - goes on github by <a href="https://github.com/Vishal0203">@Vishal0203</a></li>
+                    </ul>
+                </p>
+                <p> .. or if you didn't like it, then use your newfound knowledge of git to open an issue and contribute here on <a href="https://github.com/CuriousLearner/git-intro/">GitHub</a>.</p>
             </section>
             <section style="text-align: left;">
                 <h1>THE END</h1>
                 <p>
-                    Questions?
```
Comment (CuriousLearner) [inline index.html]: Undo this. 

May be fix email address to be sanyam@sanyamkhurana.com

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Review inline.

Thanks for your contributions.

File: README.md (hunk 1 above)
Comment (vipulgupta2048) [reply README.md]: Roger that.

File: index.html (hunk 2 above)
Comment (vipulgupta2048) [reply index.html]: That's already done. Removing this bit.

File: index.html (hunk 3 above)
Comment (vipulgupta2048) [reply index.html]: Well, I did convert all tabs to spaces. What indent are we looking at 4 or 2

File: index.html (hunk 5 above)
Comment (vipulgupta2048) [reply index.html]: So should we enforce 79 char per line? because indentation will be indentation.

File: index.html (hunk 3 above)
Comment (CuriousLearner) [reply index.html]: Ideally it would be 2, but if it's 4 everywhere now, let it be 4.

File: index.html (hunk 6 above)
Comment (vipulgupta2048) [reply index.html]: Done. Converted.
Git stores everything in its database not by file name but by the hash value of its contents. These hash values are used very frequently.

File: index.html (hunk 5 above)
Comment (CuriousLearner) [reply index.html]: Yeah for long lines not for others. Can go to 120 chars as well. There is no hard limit. But I just don't want something really long, running entire 2 screens.

File: index.html (hunk 7 above)
Comment (vipulgupta2048) [reply index.html]: Corrected.

File: index.html (hunk 8 above)
Comment (vipulgupta2048) [reply index.html]: Liked the idea. Implemented,

File: index.html (hunk 9 above)
Comment (vipulgupta2048) [reply index.html]: Done, email ID changed.

File: index.html (hunk 4 above)
Comment (vipulgupta2048) [reply index.html]: Done

File: index.html (hunk 5 above)
Comment (vipulgupta2048) [reply index.html]: Gotcha

File: index.html (hunk 3 above)
Comment (vipulgupta2048) [reply index.html]: Made indentations as 2

**Comment by vipulgupta2048** [comment]
@CuriousLearner Changes has been made as requested. Indentation has been changed to 2.

File: README.md [hunk 10]
```diff
@@ -2,6 +2,11 @@
 Learning version control with git
 
 This is currently under development.
-Created to teach people about version control using git.
+Created to teach people about version control using git and GitHub. Feel free to use it for your own purpose, by referring back to the original contributors </n>
```
Comment (CuriousLearner) [inline README.md]: Remove sentence, `Feel free to use it for your own purpose, by referring back to the original contributors </n>`

The terms are in he license.

Comment (CuriousLearner) [reply README.md]: Also, remove Github. The main focus is on teaching git.

Comment (vipulgupta2048) [reply README.md]: Removed.

**Comment by vipulgupta2048** [comment]
@CuriousLearner Ready to go.

**Review by CuriousLearner — APPROVED** [review]

**Pull request merged**